invokeai/app/services/graph.py

```diff
@@ -211,9 +212,10 @@ def invoke(self, context: InvocationContext) -> CollectInvocationOutput:
 InvocationsUnion = Union[BaseInvocation.get_invocations()]  # type: ignore
 InvocationOutputsUnion = Union[BaseInvocationOutput.get_all_subclasses_tuple()]  # type: ignore
 
-
-class Graph(BaseModel):
+class Graph(InvokeAISettings):
     id: str = Field(description="The id of this graph", default_factory=lambda: uuid.uuid4().__str__())
+    type: Literal["graph"] = "graph"
```

Comment:
What is this literal for?

Reply:
Graph inherits from BaseInvocation, which inherits from InvokeAISettings. InvokeAISettings looks for a `type` literal with the name of the node or command in order to get the appropriate configuration settings from the config file. 

Since Graph now inherits from InvokeAISettings, it should have this `type` slot set to something, so it does. In theory this could be used to configure any graph's default parameters within the init file using a `graph:` stanza. Obviously it would be better to give each graph a unique name so that they can be configured independently of each other. If you can suggest how you'd like this to work, I'll put code into the configurator such that if the type is "graph" then a second graph name attribute will be used to distinguish the individual graph.

Alternatively graphs could be exempted from the init file configuration system by reverting them back to BaseModel.

Reply:
`Graph` doesn't inherit from `BaseInvocation`, just `BaseModel` (from pydantic). `GraphInvocation` *is* a node, but that's not what you've added `type` to here.